src/Features/CSharp/Portable/Completion/CompletionProviders/OperatorsAndIndexer/UnnamedSymbolCompletionProvider_Operators.cs

```diff
@@ -0,0 +1,177 @@
+
+        static UnnamedSymbolCompletionProvider()
+        {
+            // Collect all the characters used in C# operators and make them filter characters and not commit
+            // characters. We want people to be able to write `x.=` and have that filter down to operators like `==` and
+            // `!=` so they can select and commit them.
+            using var _ = PooledHashSet<char>.GetInstance(out var filterCharacters);
+
+            for (var i = 0; i < s_operatorInfo1.Length; i++)
+            {
+                var (opName, position) = s_operatorInfo1[i];
+                var opText = GetOperatorText(opName);
+                s_operatorNameToInfo[opName] = (sortOrder: i, position);
+
+                foreach (var ch in opText)
+                {
+                    if (!char.IsLetterOrDigit(ch))
+                        filterCharacters.Add(ch);
+                }
+            }
+
+            var opCharacters = ImmutableArray.CreateRange(filterCharacters);
+            s_operatorRules = CompletionItemRules.Default
+                .WithFilterCharacterRule(CharacterSetModificationRule.Create(CharacterSetModificationKind.Add, opCharacters))
+                .WithCommitCharacterRule(CharacterSetModificationRule.Create(CharacterSetModificationKind.Remove, opCharacters));
+        }
+
+        private void AddOperatorGroup(CompletionContext context, string opName, IEnumerable<ISymbol> operators)
+        {
+            if (!s_operatorNameToInfo.TryGetValue(opName, out var sortOrderAndPosition))
+                return;
+
+            var displayText = GetOperatorText(opName);
+
+            context.AddItem(SymbolCompletionItem.CreateWithSymbolId(
+                displayText: displayText,
+                displayTextSuffix: null,
+                inlineDescription: GetOperatorInlineDescription(opName),
+                filterText: displayText,
+                sortText: SortText(OperatorSortingGroupIndex, $"{sortOrderAndPosition.sortOrder:000}"),
+                symbols: operators.ToImmutableArray(),
+                rules: s_operatorRules,
+                contextPosition: context.Position,
+                properties: OperatorProperties.Add(OperatorName, opName)));
+        }
+
+        private static string GetOperatorText(string opName)
+            => SyntaxFacts.GetText(SyntaxFacts.GetOperatorKind(opName));
+
+        private async Task<CompletionChange> GetOperatorChangeAsync(
+            Document document, CompletionItem item, CancellationToken cancellationToken)
+        {
+            var opName = item.Properties[OperatorName];
+            var opPosition = GetOperatorPosition(opName);
+
+            if (opPosition.HasFlag(OperatorPosition.Infix))
+                return await ReplaceTextAfterOperatorAsync(document, item, text: $" {item.DisplayText} ", cancellationToken).ConfigureAwait(false);
+
+            if (opPosition.HasFlag(OperatorPosition.Postfix))
```

Comment:
So increment/decrement operator will always complete as postfix (also as displayed in inline description). Not sure we make the enum a flag then, it might make the intention clearer w/o that.